### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
Deprecations
- **flutter_svg**: The color and colorBlendMode properties have been removed. Instead, use the colorFilter property.
- **mobile_scanner**:
  - allowDuplicates is removed and replaced by MobileScannerSpeed enum
  - onDetect now returns the object BarcodeCapture, which contains a List of barcodes and, if enabled, an image.

Important fixes:
- **connectivity_plus**: Do not return ConnectivityResult.none on iOS and MacOS with VPN (#1335).

### CODE REVIEW

File: lib/routes/home/widgets/app_bar/home_app_bar.dart [hunk 1]
```diff
@@ -24,7 +24,10 @@ class HomeAppBar extends AppBar {
               "src/icon/hamburger.svg",
               height: 24.0,
               width: 24.0,
-              color: themeData.appBarTheme.actionsIconTheme?.color,
+              colorFilter: ColorFilter.mode(
+                themeData.appBarTheme.actionsIconTheme!.color!,
```
Comment (ubbabeck) [inline lib/routes/home/widgets/app_bar/home_app_bar.dart:28]: Are there any chance that these can be null ?

**Review by ubbabeck — APPROVED** [review]
Tested on receiving and scanning on iOS. Sending works and receiving flow is smooth. Does not freeze anymore after receiving funds either.  Approve after addressing the null safety question.

File: lib/routes/home/widgets/app_bar/home_app_bar.dart (hunk 1 above)
Comment (erdemyerebasmaz) [reply lib/routes/home/widgets/app_bar/home_app_bar.dart:28]: Not in our case, they are defined on every theme available on our app.

Comment (ubbabeck) [reply lib/routes/home/widgets/app_bar/home_app_bar.dart:28]: Great, thank you 🙏

**Pull request merged**